commons/zenoh-config/src/lib.rs

```diff
@@ -446,25 +446,15 @@ validated_struct::validator! {
                                 /// if still no batch is available.
                                 wait_before_drop: i64,
                                 /// The maximum deadline limit for multi-fragment messages.
-                                /// When sending multi-fragment message, for each consecutive fragment the deadline from
-                                /// the start point of message transmission is calculated as
-                                /// wait_before_drop * 2 ^ min(N, ttl) where N is a fragment number.
-                                /// Thus, this parameter allows setting a limitation for maximum time for multi-fragment
-                                /// message transmission as wait_before_drop * 2 ^ ttl
-                                ttl: usize,
+                                max_wait_before_drop_fragments: i64,
```

Comment:
Are negative values allowed? It's a bit weird to me. Same thing for `wait_before_drop` and `wait_before_close`.

Reply:
Negative values are interpreted as infinity:
https://github.com/eclipse-zenoh/zenoh/blob/075f2ad30f4143104bf917246f1f90f100770c3b/io/zenoh-transport/src/manager.rs#L44

And that makes sense for some cases

Reply:
> an awful, implicit, and non-documented way of configuring infinite duration

Agree!

https://github.com/eclipse-zenoh/zenoh/issues/1614